packages/shell-chrome/assets/panel.html

```diff
@@ -241,8 +289,57 @@ <h5 class="flex items-center px-2 leading-7 font-mono">
                         </div>
                     </div>
 
+
+                    <!-- Split Pane Gutter -->
+                    <div 
+                        id="gutter" 
```

Comment:
could we use a ref instead?

Reply:
Ah I started with this in an external file...don't know why I didn't switch to a ref once I brought it into the alpine instance 🤷🏻‍♂️ Good shout!